recipes/centos_vertx/cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: vertx
```

Comment:
@l0rd @benoitf we need one for each recipe. Say you want to build a `registry.centos.org/che-stacks/vertx` and `registry.centos.org/che-stacks/springboot`, you'll have separate dirs for their Dockerfiles and each dir needs to have a relevant `cccp.yml`. 

Something like:

```
├── springboot
│   ├── cccp.yml
│   └── Dockerfile
└── vertx
    ├── cccp.yml
    └── Dockerfile
```